PlaidLink.js

```diff
@@ -114,6 +114,12 @@ PlaidLink.propTypes = {
 
   // Optional props
 
+  // You can configure Link to return only the accounts that are a
+  // checking or savings account subtype
```

Comment:
rather than "that are a checking or savings account subtype" -> "that match a given type and subtype".

Also maybe indicate this should be an object with string arrays for each account type key?